src/services/stats/stats.class.ts

```diff
@@ -54,15 +96,18 @@ export class Stats implements ServiceMethods<Data> {
     return data;
   }
 
-  async update(id: NullableId, data: Data, params?: Params): Promise<Data> {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, class-methods-use-this
+  async update(_id: NullableId, data: Data, _params?: Params): Promise<Data> {
     return data;
   }
 
-  async patch(id: NullableId, data: Data, params?: Params): Promise<Data> {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, class-methods-use-this
+  async patch(_id: NullableId, data: Data, _params?: Params): Promise<Data> {
     return data;
   }
 
-  async remove(id: NullableId, data: Data, params?: Params): Promise<Data> {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, class-methods-use-this
```

Comment:
oktu peux le mettre en haut du fichier pour tout le fichier en une fois
/* eslint-disable class-methods-use-this */
/* eslint-disable @typescript-eslint/no-unused-vars */